Alignment/CommonAlignmentProducer/plugins/PCLTrackerAlProducer.cc

```diff
@@ -507,7 +507,7 @@ ::createGeometries(const edm::EventSetup& setup)
     setup.get<PTrackerParametersRcd>().get( ptp );
 
     theTrackerGeometry = boost::shared_ptr<TrackerGeometry>(
-        trackerBuilder.build(&(*geometricDet), *ptp )
+        trackerBuilder.build(&(*geometricDet), *ptp, tTopo )
```

Comment:
@ebrondol - TrackerTopology is built from Tracker Parameters and has them cached. It's not needed to pass both.